Jucer2Reprojucer/main.cpp

```diff
@@ -1641,6 +1646,9 @@ int main(int argc, char* argv[])
                                   if (value == "-std=c++14")
                                     return "C++14";
 
+                                  if (value == "-std=c++17")
+                                    return "C++17";
+
```

Comment:
The Linux Makefile exporter setting "C++ standard to use" was replaced by the project setting "C++ Language Standard" in https://github.com/WeAreROLI/JUCE/commit/2e81e8a0dc8e4c509baa3317ad760153ddb2eeb5#diff-c0f0b20e4d0da4c5c89c5944c3506975L361-L368. At that time, Projucer didn't know about C++17 at all, so these 3 lines are actually dealing with unexpected input. Please remove them.